CONTRIBUTING.rst

```diff
@@ -124,62 +117,51 @@ Writing your first Pull Request
 Preparing the ground
 ^^^^^^^^^^^^^^^^^^^^
 
-If not already done, first create an issue, and discuss
-the changes with the project's maintainers.
-
-See in the sections above for the right way to do this.
+Before writing any code, ensure you have created an issue
+discussing the proposed changes with the maintainers.
+See the relevant sections above on how to do this.
 
 Setting up the environment
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-First, `fork skrub on Github <https://github.com/skrub-data/skrub/fork>`__.
-
-That will enable you to push your commits to a branch *on your fork*.
+Follow the steps in the :ref:`installation_instructions` > "From Source" section to
+set up your environment.
 
-Then, clone the repo on your computer:
-
-.. code:: console
-
-   git clone https://github.com/<YOUR_NAME>/skrub
-
-It is advised to create a new branch every time you work on a new issue,
-to avoid confusion:
+When starting to work on a new issue, it's recommended to create a new branch:
 
 .. code:: console
 
    git switch -c branch_name
 
-Finally, install the dependencies by heading to the `installation process <https://skrub-data.org/stable/install.html#advanced-usage-for-contributors>`__,
-advanced usage section.
-
-Implementation
-^^^^^^^^^^^^^^
 
-There are a few specific project goals to keep in mind:
+.. _implementation guidelines:
```

Comment:
I'd use
```suggestion
.. _implementation-guidelines:
```

here and in install.rst

Reply:
I use this small tweak to put the anchor as is in the text, see: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-a-custom-anchor

Reply:
Yes, I was wondering if we should use ``-`` between words when creating anchors, like in ``.. _writing-your-first-pull-request:`` L112